src/build.cc

```diff
@@ -168,6 +168,9 @@ string BuildStatus::FormatProgressStatus(
   string out;
   char buf[32];
   int percent;
+  char total_edges_str[32];
+  snprintf(total_edges_str, sizeof(total_edges_str), "%d", total_edges_);
```

Comment:
Note: total_edges_ can change over the course of a build, so this isn't guaranteed.  The most common thing in fact is for the total edge count to be overestimated and then reduced in the case where a command doesn't modify its outputs.